server/controller/src/main/java/ai/starwhale/mlops/api/TaskLogWsServer.java

```diff
@@ -41,39 +37,64 @@
 @Component
 @Slf4j
 @ServerEndpoint("/api/v1/log/online/{taskId}")
-//@ServerEndpoint("${sw.controller.apiPrefix}/log/online/{taskId}")
 public class TaskLogWsServer {
 
-    private static final ConcurrentHashMap<String, TaskLogWsServer> sockets = new ConcurrentHashMap<>();
+    private static final ExecutorService executorService = Executors.newCachedThreadPool();
 
     private static IdConvertor idConvertor;
 
+    private static CancellableTaskLogK8sCollectorFactory logCollectorFactory;
```

Comment:
`static` is not needed

Reply:
I want this factory shared between the TaskLogWsServer instances.

Reply:
`CancellableTaskLogK8sCollectorFactory` is a singleton bean held by Spring. And you have `setLogCollectorFactory` by `Autowired`. So it is shared even `static` is removed.